apache-poi/src/test/java/com/baeldung/poi/excel/write/number/NumberCellValueUnitTest.java

```diff
@@ -0,0 +1,107 @@
+package com.baeldung.poi.excel.write.number;
```

Comment:
Let's put this test class in the same package as the main class and name it ExcelNumericFormatUnitTest